allauth/socialaccount/providers/daum/provider.py

```diff
@@ -8,7 +8,7 @@ def get_avatar_url(self):
         return self.account.extra_data.get("bigImagePath")
 
     def to_str(self):
-        return self.account.extra_data.get("nickname", self.account.uid)
+        return self.account.extra_data.get("userid", self.account.uid)
```

Comment:
Given that `to_str()` is supposed o return human recognizable identifiers, I don't think `userid` nor `uid` is the best choice here. Why not keep `nickname`?

Reply:
`userid` is a string with alphanumeric characters, according to the `tests.py` file. Nicknames are not guaranteed to be unique. That's why I changed it to `userid`. I haven't tested this with a real Daum account.